aiidalab_widgets_base/process.py

```diff
@@ -261,7 +261,7 @@ def __init__(
         self,
         process=None,
         followers=None,
-        update_interval=0.1,
+        update_interval=1.0,
```

Comment:
Seems like a more reasonable default here.

Reply:
For the remote running process where the interval cooling time usually > 10s, `1.0` seems better, but we set localhost where `safe-interval=0.0` (https://github.com/aiidalab/aiidalab-docker-stack/blob/6cfb093b7dd91272258023d6d45e92da47986ddc/stack/base/before-notebook.d/40_prepare-aiida.sh#L63). So it makes sense to set to `0.1` to see the quick update of the calcjob.

Reply:
Hmm, I am not sure I agree. `0.1` can in my experience put a lot of unnecessary CPU load on the server. Also, typical AiiDA ab initio jobs will run quite long, surely not much can happen within 0.1? Note that we've already made this default decision for the `ProcessMonitor` class, so not sure it make sense to have a different default here.

Reply:
I see~ Okay, let's set it to 1.0s.